src/core/internalMemMgr.h

```diff
@@ -56,6 +56,9 @@ struct GpuMemoryPool
     uint64                          pagingFenceVal;         // Paging fence value
 
     Util::BuddyAllocator<Platform>* pBuddyAllocator;        // Buddy allocator used for the suballocation
+    bool                            mapped;                 // flag to check if this pool is already mapped
```

Comment:
I don't think you need this flag.  If `pData` is `nullptr`, then the memory is not already mapped.

Reply:
Done